src/Propel/Generator/Command/InitCommand.php

```diff
@@ -225,6 +226,27 @@ private function generateProject(OutputInterface $output, array $options)
         $this->writeFile($output, sprintf('%s/schema.xml', $options['schemaDir']), $options['schema']);
         $this->writeFile($output, sprintf('%s/propel.%s', getcwd(), $options['format']), $config->render($options));
         $this->writeFile($output, sprintf('%s/propel.%s.dist', getcwd(), $options['format']), $distConfig->render($options));
+
+        $this->buildSqlAndModelsAndConvertConfig();
+    }
+
+    private function buildSqlAndModelsAndConvertConfig()
+    {
+        $this->getApplication()->setAutoExit(false);
+
+        $followupCommands = [
```

Comment:
Could we call this `followUpCommands` (I guess the english word for this is "follow up" and thus the camel case version would be "followUp" instead of "followup")